### PR TITLE
tests: run autopkgtests in the autopkgtest.ubuntu.com infrastructure

### PR DESCRIPTION
**Pull request by mvo5**

### CODE REVIEW

**Comment by martinpitt** [comment]
FTR: Right now this always tests master (`&build-git=https://github.com/snapcore/snapd`) -- please just drop that argument. Then https://git.launchpad.net/~ubuntu-release/+git/autopkgtest-cloud/commit/?id=c62a5cb67ec will kick in and generate the correct clone URL for that PR automatically.

File: tests/main/refresh-undo/task.yaml [hunk 1]
```diff
@@ -15,17 +15,32 @@ prepare: |
     snapbuild $TESTSLIB/snaps/$SNAP_NAME_GOOD .
     snapbuild $TESTSLIB/snaps/$SNAP_NAME_BAD .
 
+debug: |
+    journalctl -u snap.test-snapd-service.service.service
```
Comment (zyga) [inline tests/main/refresh-undo/task.yaml]: Was the bork bork bork intentional?

Comment (mvo5) [inline tests/main/refresh-undo/task.yaml]: Not really, no :)

Comment (mvo5) [inline tests/main/refresh-undo/task.yaml]: But it is the correct name of the snap service (because the service itself is called "service").

**Comment by martinpitt** [comment]
@mvo5: Niice! :+1:

**Pull request merged**